airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -1686,6 +1687,7 @@ private Set<String> getConnectorRepositoriesInUse(final DSLContext ctx) {
   Map<String, ConnectorInfo> getConnectorRepositoryToInfoMap(final DSLContext ctx) {
     return ctx.select(asterisk())
         .from(ACTOR_DEFINITION)
+        .where(ACTOR_DEFINITION.RELEASE_STAGE.isNull().or(ACTOR_DEFINITION.RELEASE_STAGE.ne(ReleaseStage.custom)))
```

Comment:
Should we also check the custom column here?

Reply:
I guess it doesn't hurt. The check could be either `(release_stage is null or release_stage <> 'custom' or custom = true)`